framework/src/fvkernels/FVFluxKernel.C

```diff
@@ -159,7 +175,7 @@ FVFluxKernel::computeJacobian(const FaceInfo & fi)
 
   _face_info = &fi;
   _normal = fi.normal();
-  const DualReal r = fi.faceArea() * fi.faceCoord() * computeQpResidual();
+  const ADReal r = fi.faceArea() * fi.faceCoord() * computeQpResidual();
```

Comment:
Are these different, or is this just a style change?

Reply:
They use to be different, but now they are the same ever since Alex slowed AD down :-P